plugin/src/index.ts

```diff
@@ -6,18 +7,20 @@ import { version } from '../package.json';
 
 import { loadDevEnvVariables } from './dev';
 import { starMatch } from './starMatch';
-import type { BuildParams } from './types';
+import type { PluginInputs } from './types';
 
 function createSummaryLogger(
-  show: BuildParams['utils']['status']['show']
+  show: NetlifyPluginUtils['status']['show']
 ): (message: string) => void {
   return (message): void => {
     show({ title: 'Algolia Crawler', summary: message });
     console.log(message);
   };
 }
 
-export async function onSuccess(params: BuildParams): Promise<void> {
+export async function onSuccess(
```

Comment:
The type of `onSuccess` itself is also exported as `OnSuccess`. One could do:

```ts
export const onSuccess: OnSuccess = async function(params) {
```

However, the `algolia` ESLint rules forbids this function declaration pattern, so this PR is not using it.